contracts/ExampleToken.cdc

```diff
@@ -1,53 +1,40 @@
 import FungibleToken from "./FungibleToken.cdc"
+import MetadataViews from "./utilityContracts/MetadataViews.cdc"
+import FungibleTokenMetadataViews from "./FungibleTokenMetadataViews.cdc"
 
 pub contract ExampleToken: FungibleToken {
 
     /// Total supply of ExampleTokens in existence
     pub var totalSupply: UFix64
-
+    
     /// Storage and Public Paths
     pub let VaultStoragePath: StoragePath
     pub let ReceiverPublicPath: PublicPath
     pub let BalancePublicPath: PublicPath
+    pub let ResolverPublicPath: PublicPath
```

Comment:
Why define a new path for the resolver? You can just put the resolve view functions in the balance interface and add default implementations now

Reply:
That's how I was doing it at the beginning (actually I was using the receiver path) but just when writing the transactions it seemed to me that I'd be easy to understand how it worked with an extra, a little unnecessary,  path. 
Actually I just realised that I forgot to use it in the `init()` function of the contract
```cadence
        // Create a public capability to the stored Vault that exposes
        // the `deposit` method through the `Receiver` interface and also
        // the `resolveView`method through the MetadataViews `Resolver` interface
        self.account.link<&{FungibleToken.Receiver, MetadataViews.Resolver}>(
            self.ReceiverPublicPath,
            target: self.VaultStoragePath
        )
```
Any particular reason it would be better to use the balance path rather than the receiver path?

Reply:
Balance is a piece of metadata, so it makes sense that it would be the one that has the rest of the metadata.

Reply:
what would you think about renaming the `ExampleToken.BalancePublicPath` as `ExampleToken.MetadataPublicPath` and expose both Balance and Resolver there?

Reply:
I'm not sure how I feel about renaming the `BalancePublicPath` to `MetadataPublicPath`. Given that neither makes perfect sense, I'd be inclined to stick with the one that's already in use.

Reply:
If we are going to change the suggested name, something like `VaultPublicPath` makes more sense to me, since that can be used to describe the "publicly available information for this vault".

Reply:
I'd prefer `VaultPublicPath` over Balance if we are exposing also the Metadata there, so I'll go with that.